overseerr-api.yml

```diff
@@ -2087,9 +2087,6 @@ paths:
                     id:
                       type: string
                       example: job-name
-                    name:
-                      type: string
-                      example: A Job Name
```

Comment:
No reason to remove this. It's also a breaking API change. Let's just leave it in.

Reply:
I didn't think it was being used anywhere, but that's a good point.  I'll add it back!

Reply:
Other people might be using it though. Never know.

Reply:
@sct Yeah you're totally right :)